.gitignore

```diff
@@ -8,3 +8,5 @@
 .vscode
 build
 CMakeLists.txt.user
+tags
```

Comment:
Add a comment about the tool that generates these (ctags and cscope?).

Reply:
Done.